src/Admin/Views/ProviderOrganizations/AddExisting.cshtml

```diff
@@ -0,0 +1,98 @@
+@model ProviderOrganizationViewModel
+@using Bit.SharedWeb.Utilities
+
+@{
+    ViewData["Title"] = "Add Existing Organization";
+}
+
+<h1>Add Existing Organization</h1>
+<div class="row mb-2">
+    <div class="col">
+        <form class="form-inline mb-2" method="get">
+            @Html.HiddenFor(m => m.ProviderId)
+            <label class="sr-only" asp-for="OrganizationName">Organization Name</label>
+            <input type="text" class="form-control mb-2 mr-2 flex-fill" placeholder="Organization Name" asp-for="OrganizationName" name="name">
+            <label class="sr-only" asp-for="OrganizationOwnerEmail">Owner email</label>
+            <input type="email" class="form-control mb-2 mr-2 flex-fill" placeholder="Owner email" asp-for="OrganizationOwnerEmail" name="ownerEmail">
+            <button type="submit" class="btn btn-primary mb-2" title="Search" formmethod="get"><i class="fa fa-search"></i> Search</button>
+        </form>
+    </div>
+</div>
+
+<form method="post" id="select-form">
+    <div class="table-responsive">
+        <table class="table table-striped table-hover">
+            <thead>
+            <tr>
+                <th style="width: 20px;">All</th>
+                <th>Name</th>
+                <th style="width: 190px;">Plan</th>
+                <th style="width: 190px;">Owner email</th>
+            </tr>
+            </thead>
+            <tbody>
+            @if (!Model.Items.Any())
+            {
+                <tr>
+                    <td colspan="5">No results to list.</td>
+                </tr>
+            }
+            else
+            {
+                @for (var i = 0; i < Model.Items.Count; i++)
```

Comment:
Suggestion: use a `foreach` loop instead, as in the organizations `Index.cshtml`. Easier to read rather than C style loops.

```cshtml
@foreach(var org in Model.Items)
```

Reply:
I completely agree that a `foreach` loop would look better but I can't get MVC to work with it - doesn't bind the values properly on the `POST` action.

Reply:
I tried it locally and it seemed to work fine. We also do it this way in some other views. Can you double check you updated all the `Model.Items[i]` to refer to the new temp variable? If it's still not working, please push your changes anyway and I'll have a look.

Reply:
I could't get it working, I updated all the references to the new temp variable:
```
@foreach (var organization in Model.Items)
{
    <tr>
        <td class="text-center">
            @Html.HiddenFor(m => organization.Id)
            @Html.CheckBoxFor(m => organization.Selected)
        </td>
        <td>@Html.ActionLink(organization.Name, "Edit", "Organizations", new { id = organization.Id }, new { target = "_blank" })</td>
        <td>@(organization.PlanType.GetDisplayAttribute()?.Name ?? organization.PlanType.ToString())</td>
    </tr>
}
```

An error is thrown on the POST action:
```
System.NullReferenceException: Object reference not set to an instance of an object.
   at Bit.Admin.Models.PagedModel`1.get_NextPage()
```